doc/experiment.rst

```diff
@@ -211,6 +211,103 @@ An ``Ensemble`` supports key features, including methods to:
 
 Visit the respective links for more information on each topic.
 
+==============
+File Structure
+==============
+When a user executes an ``Experiment`` script, it generates output folders in the system's directory.
+By default, SmartSim creates a predefined file structure and assigns a path to each entity initialized.
+However, users have the flexibility to customize this according to workflow needs. Please refer
+to the respective :ref:`default<default_folder>` and :ref:`configure<config_folder>` sections below
+for more details.
+
+.. _default_folder:
+
+Default
+=======
+By default, an ``Experiment`` folder is created in your current working directory, using the
+specified `name` parameter during ``Experiment`` initialization. Each entity created by the
+``Experiment`` generates an output folder under the ``Experiment`` directory, named after the
+entity. These folders hold `.err` and `.out` files, containing execution-related information.
```

Comment:
Should we call out that files that were added to be symlinked, copied, and/or configured will not be placed in these directories unless experimernt.generate() is called with the entity?

Reply:
100% Ill add